### PR TITLE
refactor(console): change label for response template type

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/8263

**Description**

https://graviteeio.slack.com/archives/C03N0JNGF62/p1659343850242819

**Additional context**

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/console-wording/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-xphdbsxzcz.chromatic.com)

### CODE REVIEW

**Comment by ctschacher** [comment]
https://github.com/gravitee-io/issues/issues/8263

File: gravitee-apim-console-webui/src/management/api/proxy/general/response-templates/response-template-type.html [hunk 1]
```diff
@@ -22,13 +22,13 @@ <h2>Template</h2>
...
-    <!-- Content-Type -->
+    <!-- Accept type -->
     <div flex-xs flex="50">
       <md-input-container class="md-block">
-        <label>Content-type</label>
+        <label>Request Accept header type</label>
         <input ng-model="$ctrl.template.type" type="text" required />
       </md-input-container>
-      <div class="hint">The content-type for which this template response must be send.</div>
+      <div class="hint">The request accept header type for which this template response must be send.</div>
```
Comment (ctschacher) [inline gravitee-apim-console-webui/src/management/api/proxy/general/response-templates/response-template-type.html]:
```suggestion
      <div class="hint">The request accept header type for which this template response should be sent.</div>
```
IMO 'should' sounds better than 'must' here and the 'send' should have a 't' instead.

**Review by ctschacher — APPROVED** [review]
LGTM

File: gravitee-apim-console-webui/src/management/api/proxy/general/response-templates/response-template-type.html (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/api/proxy/general/response-templates/response-template-type.html]: Thank's

**Pull request merged**